lldb/tools/lldb-vscode/README.md

```diff
@@ -212,6 +212,21 @@ This loads the coredump file `/cores/123.core` associated with the program
 }
 ```
 
+### Connect to a Remote Debug Server
+
+This connects to a debug server (e.g. `lldb-server`, `gdbserver`) that is
+debugging the program `/tmp/a.out` and listening locally on port `2345`.
+
+```javascript
+{
+  "name": "Remote Debug Server",
```

Comment:
we are specifying the name "Remote Debug Server", but below we are attaching locally. It might be nice to have this either save "Attach to local GDB remote server", or leave the name the same and then change the "gdb-remote 2345" to "gdb-remote other.host.com:2345"

Reply:
We could also add two entries into this:
```
### Connect to a GDB server on current machine
...
### Connect to a remote GDB server on another machine
```

Reply:
I second the idea of showing the two cases

Reply:
Changed to a local and a remote example.